iothub/device/devdoc/API Design.md

```diff
@@ -0,0 +1,255 @@
+## Plug and Play convention compatible APIs
+
+### Properties
+
+```csharp
+/// <summary>
+/// Update a single property.
+/// 
+/// Property under the root component:
+/// "reported": {
+///     "temperature": 21.3
+/// }
+/// 
+/// Property under a specified component:
+/// "reported": {
+///     "thermostat1": {
+///         "__t": "c",
+///         "temperature": 21.3
+///     }
+/// }
+/// </summary>
+/// <param name="propertyName">The name of the twin property to update.</param>
```

Comment:
We should not use the word "twin" anywhere relating to properties, since the app developer shouldn't care.

Reply:
good point, noted.